packages/teams-js/src/public/runtime.ts

```diff
@@ -154,6 +160,10 @@ export const versionConstants: Record<string, Array<ICapabilityReqs>> = {
     },
   ],
   '2.0.1': [
+    {
```

Comment:
I don't think we need this anymore because geoLocation isn't called by location at all we don't need a default isSupported value.